index.html

```diff
@@ -2979,6 +2979,15 @@ <h2 id="notes">Notes</h2>
         </ul>
       </p>
 
+      <p id="relational-operator-note">
+        <b class="header">Underscore version</b>
+        <br>
+        It's possible to get current Underscore version via
+        <tt>_.VERSION</tt> 
+        .
+        <pre>_.VERSION => 1.13.7</pre>
```

Comment:
The characters `<`, `>`, `&`, `"` and `'` need to be escaped even within `<pre>`.

```suggestion
        <pre>_.VERSION =&gt; 1.13.7</pre>
```